app/src/main/java/org/oppia/app/activity/ActivityComponent.kt

```diff
@@ -63,6 +64,7 @@ interface ActivityComponent {
   fun inject(adminAuthActivity: AdminAuthActivity)
   fun inject(administratorControlsActivity: AdministratorControlsActivity)
   fun inject(adminPinActivity: AdminPinActivity)
+  fun inject(appVersionActivity: AppVersionActivity)
```

Comment:
Reordering has been done incorrectly. It should come below `AppLanguageActivity`

Reply:
Done